site/content/docs/main/configuration.md

```diff
@@ -180,6 +180,7 @@ The gateway configuration block is used to configure which gateway-api Gateway C
 
 | Field Name | Type| Default  | Description |
 |------------|-----|----------|-------------|
+| controllerName | string |  | Gateway Class controller name (i.e. projectcontour.io/projectcontour/contour).  |
```

Comment:
```suggestion
| controllerName | string | `projectcontour.io/<contour namespace>/contour` | Specifies the GatewayClass controller name to reconcile. |
```

maybe add this as the default? If i remember right that's what the default ends up being, maybe good enough to make the namespace `projectcontour`

Reply:
There isn't a default anymore. The controllerName was made a pointer right now to allow some time to not break any existing users.

Reply:
ah ok, disregard then